src/erasure-code/jerasure/CMakeLists.txt

```diff
@@ -31,7 +31,7 @@ add_library(ec_jerasure_generic SHARED
   $<TARGET_OBJECTS:erasure_code_objs>
   )
 add_dependencies(ec_jerasure_generic ${CMAKE_SOURCE_DIR}/src/ceph_ver.h)
-target_link_libraries(ec_jerasure_generic ${EXTRALIBS})
+target_link_libraries(ec_jerasure_generic crush ${EXTRALIBS})
```

Comment:
i don't understand why the erasure code libs would depend on crush. if you're getting linker errors without this, we should figure out what's pulling crush stuff in